api/tests/unit/infrastructure/serializers/csv/csv-serializer_test.js

```diff
@@ -0,0 +1,65 @@
+const { expect } = require('../../../../test-helper');
+
+const csvSerializer = require('../../../../../lib/infrastructure/serializers/csv/csv-serializer');
+
+describe('Unit | Serializer | CSV | csv-serializer', () => {
+
+  describe('#serializeLine', () => {
+
+    it('should quote strings', async () => {
```

Comment:
les `async` semblent inutiles : aucun `await` et on ne retourne pas de promesse.
```suggestion
    it('should quote strings', () => {
```